src/js/helpers/covid19Helper.js

```diff
@@ -164,10 +164,13 @@ export const getTotalSpendingAbbreviated = (totalSpending) => {
     return `${abbreviatedValue} ${unit.longLabel}`;
 };
 
-export const areCountsDefined = (counts) => Object.keys(counts).reduce((acc, tab) => {
-    if (acc === null) return acc;
-    return counts[tab];
-}, true);
+export const areCountsDefined = (counts) => {
+    const countTabs = Object.keys(counts);
+    if (countTabs.length === 0) {
+        return false;
+    }
+    return countTabs.reduce((acc, tab) => acc && counts[tab] !== null, true);
```

Comment:
Can we add a test case to cover the condition where this was returning `falsy`? This would just mean adding an `it` block to the existing tests where one of the tab counts is zero; preferably the last property in the tabCounts object like

```javascript
const mockTabCounts = { test1: 5, test2: 1, test3: 0 }
```

Reply:
Will do